src/app/grid/grid-routes-data.ts

```diff
@@ -106,5 +106,6 @@ export const gridsRoutesData = {
     "remote-paging-batch-editing": {displayName: "Remote paging with batch editing", parentName: "Grid"},
     "grid-sorting-indicators": {displayName: "Grid Sorting Indicators", parentName: "Grid"},
     "exporting-visualization": {displayName: "Exporting Visualization", parentName: "Grid"},
-    "grid-editing-lifecycle": {displayName: "Grid editing lifecycle", parentName: "Grid"}
+    "grid-editing-lifecycle": {displayName: "Grid editing lifecycle", parentName: "Grid"},
+    "grid-select": {displayName: "Grid with Select", parentName: "Grid"}
```

Comment:
It will be better to rename "Grid with Select" to something more informative like "Grid With Select Editor Provider"